### ISSUE TITLE
Main.py

### ISSUE THREAD

**Issue by Srglowstone** [Srglowstone/IA_Template#1]
import tensorflow as tf
import numpy as np

# Generamos los datos de entrenamiento
x_train = np.array([[1, 1, 1], [2, 2, 1], [3, 3, 1], [4, 4, 1], [5, 5, 1], [5, 10, 1], [10, 10, 1], [25, 25, 1], [1, 1, -1], [2, 2, -1], [3, 3, -1]], dtype=float)
y_train = np.array([2, 4, 6, 8, 10, 15, 20, 50, 0, 0, 0], dtype=float)

# Creamos un modelo secuencial con una sola capa oculta de tres unidades y una capa de salida de una unidad
model = tf.keras.models.Sequential()
model.add(tf.keras.layers.Dense(3, input_shape=(3,), activation='relu'))
model.add(tf.keras.layers.Dense(1, activation='linear'))

# Compilamos el modelo
model.compile(
    optimizer=tf.keras.optimizers.Adam(0.01),
    loss='mean_squared_error')

# Entrenamos el modelo
print("Entrenando modelo.")
history = model.fit(x_train, y_train, epochs=1000, verbose=False)
print("Modelo entrenado.")

# Hacemos una predicción
operation = 1
result = model.predict([[5, 5, operation]])
if operation == 1:
  print("El resultado de la suma es: ", result[0][0])
else:
  print("El resultado de la resta es: ", result[0][0])
